docker/files/run/run.sh

```diff
@@ -182,19 +201,21 @@ function start_new_private_network() {
 ##############
 
 echo "Starting Algod Docker Container"
-echo "   ALGORAND_DATA:  $ALGORAND_DATA"
-echo "   NETWORK:        $NETWORK"
-echo "   PROFILE:        $PROFILE"
-echo "   DEV_MODE:       $DEV_MODE"
-echo "   START_KMD:      ${START_KMD:-"Not Set"}"
-echo "   FAST_CATCHUP:   $FAST_CATCHUP"
-echo "   TOKEN:          ${TOKEN:-"Not Set"}"
-echo "   ADMIN_TOKEN:    ${ADMIN_TOKEN:-"Not Set"}"
-echo "   KMD_TOKEN:      ${KMD_TOKEN:-"Not Set"}"
-echo "   TELEMETRY_NAME: $TELEMETRY_NAME"
-echo "   NUM_ROUNDS:     $NUM_ROUNDS"
-echo "   PEER_ADDRESS:   $PEER_ADDRESS"
-echo "   ALGOD_PORT:     $ALGOD_PORT"
+echo "   ALGORAND_DATA:   $ALGORAND_DATA"
+echo "   NETWORK:         $NETWORK"
+echo "   PROFILE:         $PROFILE"
+echo "   DEV_MODE:        $DEV_MODE"
+echo "   START_KMD:       ${START_KMD:-"Not Set"}"
+echo "   FAST_CATCHUP:    $FAST_CATCHUP"
+echo "   TOKEN:           ${TOKEN:-"Not Set"}"
+echo "   ADMIN_TOKEN:     ${ADMIN_TOKEN:-"Not Set"}"
+echo "   KMD_TOKEN:       ${KMD_TOKEN:-"Not Set"}"
+echo "   TELEMETRY_NAME:  $TELEMETRY_NAME"
+echo "   NUM_ROUNDS:      $NUM_ROUNDS"
+echo "   GENESIS_ADDRESS: $GENESIS_ADDRESS"
+echo "   PEER_ADDRESS:    $PEER_ADDRESS"
+echo "   GOSSIP_PORT:     $GOSSIP_PORT"
```

Comment:
This port could be unexposed if it isn't set to 4160?

Reply:
4160 is the REST API, the gossip port is the one used for consensus.

Reply:
isn't 8080 is the default REST API port and 4160 gossip port? https://github.com/algorand/go-algorand/pull/5525/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L60
if the gossip port isn't exposed, how does a nonrelay node  container connect to a relay container?

Reply:
I'm not sure if there are default, `algod.net` in the data dir has the REST API address and `algod-listening.net` has the gossip port (when enabled)

Reply:
confirmed locally that the gossip port can be unexposed.